samples/detect.v1p2beta1.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function detectPdfText(bucketName, fileName) {
+  // [START vision_async_detect_document_ocr]
+
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p2beta1;
+
+  // Creates a client
+  const client = new vision.ImageAnnotatorClient();
+
+  const gcsSourceUri = `gs://${bucketName}/${fileName}`;
+  const gcsDestinationUri = `gs://${bucketName}/${fileName}.json`;
+
+  console.log(gcsDestinationUri);
```

Comment:
Is this leftover from debugging or intentionally being printed? I don't think it's necessary to print. If so, add additional text to provide context, eg.

```js
log(`JSON will be saved to: ${gcsDestinationUri}`);
```

But I vote not printing anything here :)

Reply:
Good thinking. I removed it.